### ISSUE TITLE
As a user, I would like to see all the Christmas tree cutting rules of a National Forest on one page so I can easily view, acknowledge and print the rules before buying my permit.

### ISSUE THREAD

**Issue by bryanmcfadden** [nciinc/fs-permit-platform#428]
This comes from SME/User usability testing from sprints 06/08.

**Comment by bryanmcfadden** [comment]
Proposal:

1. Change existing buy a permit page to user information intake and limited information only. (remove rules and user acknowledgement) Change button text to 'Continue'.

![sp_09_rules_separate_page_proposal_page_01_flathead](https://user-images.githubusercontent.com/16689692/36399041-12878828-1598-11e8-8114-a77bccfc0703.jpg)

2. Create a new second 'Christmas tree cutting rules' page which contains rules (from markdown file of forest) that can be printed. User acknowledgement is then added to bottom of rules container with button text change to "Continue to Checkout'.

![sp_09_rules_separate_page_proposal_page_02_flathead](https://user-images.githubusercontent.com/16689692/36399045-173830de-1598-11e8-988f-4af54aff1b81.jpg)

**Comment by ASprinkle** [comment]
I really like these ideas. I especially like the "next" button for the permittee information, so that it's a standalone page with the "print" button.  I am interested in seeing how step 3 would come together for each NF.  I do like the idea of the rules being embedded throughout the page, and then "one-stop-shop" at the end.

**Comment by MelissaBraxton** [comment]
Agree with @ASprinkle - This is a great direction! I like the current mocks (although the content of the standalone rules page needs some work, but that's to be expected at this early stage)

We will need to think through what users will see/do if, after reading the rules, they decide not to continue to check out. 

We may also want to consider showing users their progress through all the steps required to complete the permit purchase process, especially with the addition of the rules page as a separate step. But this doesn't all need to be handled as part of this story. 

I say go forth!

**Comment by sadlerw** [comment]
@MelissaBraxton note that 474 is for the first two mocks here for arp and this story is just for the third mock (the tree-guidelines page).  I like the idea of a wizard breadcrumb on the application, but maybe we could just use the existing breadcrumb for that?  Please make a story if you think so, or for using a new breadcrumb just for the application wizard.

Right now they can use the back button to go back from the rules on the application, but the addition of a cancel button might be necessary - if so please put a comment on 474.  

thanks! Will

**Comment by npage123** [comment]
@ASprinkle @MelissaBraxton @tram  - Here's the proposed updated guidelines page. Please take a look and provide feedback. Note: the tree species content would follow the cutting tips as seen in the left nav.
![428_1](https://user-images.githubusercontent.com/12956624/36604678-fba5810c-1883-11e8-8ba5-4c99203bb921.jpg)

**Comment by tram0004** [comment]
Needs to be rewritten using metric system to comply with new federal compliance rules.

**Comment by sadlerw** [comment]
You are a funny guy @tram004 - I'll tell Nate to use the right one! :-)

**Comment by trammell** [comment]
Needs to include guidance on ear plugs to use to muffle the noise of the trees screaming during the trimming process.

**Comment by ASprinkle** [comment]
It looks like the story is for the ARNF, is it an issue if the above image indicates the Flathead NF?  
I assume the ask for the review is to just look at the overall flow and design (which I think looks great and is easy to read), and to not focus on content at the moment, correct?
If content review is part of the ask I will have a lot of comments.

**Comment by bryanmcfadden** [comment]
@ASprinkle - You are correct...the design was created using Flathead as the example NF...but then the story was changed to reflect the Arapaho-Roosevelt NF...so the content hasn't been updated (or in the mockups) to reflect this story.

**Comment by tram** [comment]
These measurement depictions are the bee's knees. LGTM!

**Comment by npage123** [comment]
Here you go @sadlerw - 
[428-icons.zip](https://github.com/nciinc/fs-permit-platform/files/1759161/428-icons.zip)

**Comment by JaneZC** [comment]
@ASprinkle    The rules for ARP are on staging here - https://forest-service-trees-staging.app.cloud.gov/christmas-trees/forests/arp/applications#rules . The year has been removed, per our conversation on Friday.

Would it be easier to edit if I provide a document to edit? Or, should folks just post content changes directly to the story? Bryan and I will also touch base with the ARP folks again to see if they have any remaining changes.

**Comment by sadlerw** [comment]
@JaneZC I think this comment was meant to be on #474?  I'm not sure what content changes you are referring to? From Amber? I think the idea here is that content changes can be made through the google forms process and we only have to get things into a shape that can be usability tested and won't require a massive re-design based on the content changes that are requested.

**Comment by JaneZC** [comment]
@sadlerw Yup! You're right; I got my stories mixed up. Adding my comment to correct story now ....

**Comment by npage123** [comment]
@sadlerw - Here's that updated tree stump icon.
[tree-stump-height-icon.svg.zip](https://github.com/nciinc/fs-permit-platform/files/1760102/tree-stump-height-icon.svg.zip)

**Comment by bryanmcfadden** [comment]
Updated user flow diagram -
Pending Melissa/Mark's remarks
[user_task_flow.pdf](https://github.com/nciinc/fs-permit-platform/files/1763029/user_task_flow_permit_purchase_master.pdf)

![user_task_flow_permit_purchase_master](https://user-images.githubusercontent.com/16689692/36730888-effeb6a0-1b96-11e8-83b3-4750d4978a04.jpg)

**Comment by ASprinkle** [comment]
Including the illustrations in this story (and for the other NFs) addresses story #95 which I will now close.

**Comment by npage123** [comment]
Some how to plan your trip icons to help call out three main bullet points we want to bring attention to and a few different layouts. For the second layout Bryan found a reference on how that could be layed out using markdown.
![428-2-01](https://user-images.githubusercontent.com/12956624/36752807-cd7f6c40-1bc9-11e8-8f24-c04b53c93785.jpg)
![428-2-02](https://user-images.githubusercontent.com/12956624/36752828-d7298af0-1bc9-11e8-84f9-176f8bf541d7.jpg)

**Comment by ASprinkle** [comment]
Edit intro paragraph: "Combined with any _combination_ local activities, visitors can make a full-day" remove duplicate "combination" word.
Edit Ranger District Headers under Where to Cut your Tree, to add "Ranger District" after Canyon Lakes and After Sulphur.
Under helpful cutting tips: instead of "tools you will need", try "Suggested tools include a..."
Under how to plan your trip: 4th bullet. Remove the "B", last bullet - I thought there was a pet restriction for the ARNF?
Contact Us:  Why list the phone number twice for offices? can the "call first" language be added on next to the phone number?

**Comment by JaneZC** [comment]
@ASprinkle The pet restriction is a 'strong recommendation' not a true prohibition, if that makes sense. That was how it was explained so it wasn't added as a 'rule.'

The Contact Us section needs some more work still, in my opinion, that goes beyond the 'rules.'  For some reason, I thought there was a separate story just for that section, but I am not seeing it? Should I create a separate story for that or add to the story 425?

**Comment by ASprinkle** [comment]
Ok, makes sense - strong recommendation.
contact us: separate story - i will look at structuring it if it doesn't already exist somewhere. Thanks!

**Comment by tram** [comment]
👍 Flow diagram looks good to me. Also contains life goals.
<img width="503" alt="screen shot 2018-02-28 at 5 13 11 pm" src="https://user-images.githubusercontent.com/43267/36816200-1e25c9fa-1cab-11e8-911a-8c55e2aa1c7c.png">
